types/connect-route/index.d.ts

```diff
@@ -0,0 +1,122 @@
+import { type NextFunction, type NextHandleFunction } from "connect";
+import * as connect from "connect";
+import { type ServerResponse } from "http";
+
+declare namespace ConnectRoute {
+    class IncomingMessage extends connect.IncomingMessage {
```

Comment:
The class declaration cannot be in this namespace as it can only contain types but this is a class

Reply:
I see, I went ahead and moved the class definition into a `declare class` outside of the namespace. It still seems to compile and test, so let me know if there's anything else with this class definition.